cmd/notation/registry.go

```diff
@@ -91,6 +93,9 @@ func getAuthClient(opts *SecureFlagOpts, ref registry.Reference) (*auth.Client,
 		},
 		Cache:    auth.NewCache(),
 		ClientID: "notation",
+		Client: &http.Client{
+			Transport: trace.NewTransport(http.DefaultTransport),
+		},
```

Comment:
Can we add trace transport only if `--debug` is toggled?

Reply:
Updated.